libr/util/str.c

```diff
@@ -272,6 +272,25 @@ R_API char *r_str_home(const char *str) {
 	return dst;
 }
 
+R_API char *r_str_r2_prefix(const char *str) {
+	const char *prefix = r_sys_prefix (NULL);
+	size_t length;
+	length = strlen (prefix) + 1;
+	if (str) {
+		length += strlen (R_SYS_DIR) + strlen (str);
+	}
+	char *dst = (char *)malloc (length);
+	if (!dst) {
+		return NULL;
+	}
+	strcpy (dst, prefix);
+	if (str) {
+		strcat (dst, R_SYS_DIR);
```

Comment:
strcat is slow and unnecessray, you have computed the length of prefix already

Reply:
Does anything speak against using a simple '/' instead of R_SYS_DIR? (We're appending regular slashes later anyway)
That would simplify the code a bit.

Reply:
Or I'll just use r_str_newf